modules/apps/frontend-editor/frontend-editor-ckeditor-web/src/main/resources/META-INF/resources/editor/BalloonEditor.js

```diff
@@ -20,11 +20,30 @@ import {Editor} from './Editor';
 import '../css/main.scss';
 
 const BalloonEditor = ({config = {}, contents, name, ...otherProps}) => {
+	const defaultExtraPlugins = 'balloontoolbar,floatingspace';
+
 	const [cssClass, setCssClass] = useState('');
 
+	const extraPlugins = config.extraPlugins ? `${config.extraPlugins},` : '';
+
+	const basicToolbars = {
+		toolbarImage: 'JustifyLeft,JustifyCenter,JustifyRight',
+		toolbarLink: 'Link,Unlink',
+		toolbarText:
+			'Bold,Italic,Underline,BulletedList,NumberedList,Link' +
+			'JustifyLeft,JustifyCenter,JustifyRight,RemoveFormat',
+	};
+
+	const editorConfig = {
+		...basicToolbars,
+		...config,
+		extraAllowedContent: '*',
+		extraPlugins: `${extraPlugins}${defaultExtraPlugins}`,
+	};
```

Comment:
@ambrinchaudhary sorry hadn't see this ... so forget [my previous comment](https://github.com/liferay-frontend/liferay-portal/pull/984#pullrequestreview-640696008).
Let me fetch this branch to see everything in context (having multiple commits didn't make this easy to see as I was reviewing them one at a time) and play around with it, before I give more feedback on the configuration.